File: Tests/Oracle/corpus/sample.clj
(ns demo.core)

;; The binding-vector case locals.scm unrolls by pair index.
(defn total [values]
  (let [sum 0
        doubled (map #(* 2 %) values)]
    (reduce + sum doubled)))

(def limit 10)

(defn shout [name]
  (clojure.string/upper-case name))

File: Tests/Oracle/corpus/sample.py
import os
from typing import List


class Widget:
    """Indentation-based blocks, not braces."""

    def __init__(self, name: str) -> None:
        self.name = name

    def shout(self) -> str:
        return self.name.upper()


def total(values: List[int]) -> int:
    result = 0
    for value in values:
        result += value
    return result
